AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -461,8 +463,10 @@ - (void)endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL))completion
     }];
     [_pendingEditCommandBlocks removeAllObjects];
 
-    [_mainSerialQueue performBlockOnMainThread:^{
-      [_delegate dataController:self endUpdatesAnimated:animated completion:completion];
+    [_editingTransactionQueue addOperationWithBlock:^{
```

Comment:
Can you add a comment for why this is necessary, in an intuitive way, In terms of describing a scenario in which it is important to go through the editing transaction queue?

Is there any reason in particular that at the top of this method you use the approach of waiting, where is here use the approach of scheduling?